scripts/src/javatests/org/oppia/android/scripts/testfile/TestFileCheckTest.kt

```diff
@@ -125,6 +128,32 @@ class TestFileCheckTest {
     assertThat(outContent.toString().trim()).isEqualTo(TEST_FILE_CHECK_PASSED_OUTPUT_INDICATOR)
   }
 
+  @Test
+  fun testTestFileCheck_minCoverageExemptedFile_notExemptedFromTestFileCheck() {
+    val testFileExemptions = TestFileExemptions.newBuilder()
+      .addTestFileExemption(
+        TestFileExemption.newBuilder()
+          .setExemptedFilePath("test/TestExemptedFile.kt")
+          .setTestFileNotRequired(true)
+      )
+      .addTestFileExemption(
+        TestFileExemption.newBuilder()
+          .setExemptedFilePath("coverage/CoverageTestFile.kt")
+          .setOverrideMinCoveragePercentRequired(5)
+      )
+      .build()
+
+    val coverageTestExemptiontextProto = tempFolder.newFile("test_exemption.pb")
+    coverageTestExemptiontextProto.outputStream().use { (testFileExemptions.writeTo(it)) }
+
+    val result = TestFileCheck(
+      "${tempFolder.root}",
+      "${tempFolder.root}/test_exemption"
+    ).getTestFileExemptionList("${tempFolder.root}/test_exemption")
+
+    assertEquals(listOf("test/TestExemptedFile.kt"), result)
```

Comment:
Use Truth.assertThat(); we don't use assertEquals().

Reply:
Okay Ben. I instead replaced it with `assertThat` checks.
Would this practice also apply to sample test cases that I created in `TestBazelWorkspaceTest`, `BazelClientTest`, and other utility Tests that I introduced recently. I kinda utilized assertEquals extensively over there! [[commit](https://github.com/oppia/oppia-android/pull/5423/files#diff-e50d7ab1968e374641f7298e27dca62ec182a42dc06dac4c47857cd7526de00fR407-R419)]

Reply:
I think it's fine there since it avoids extra dependency complexity in setting up the tests (which is why I didn't comment on it); feel free to keep those as-is.